data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -136,6 +137,24 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         {"request":"/index.html","aggr._max":0.55,"aggr._min":0.15,"aggr._buckets":[0.0, 0.25, 0.5],"sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "aggr._bucket_counts":[0,2,1,1],"aggr._count":4,"aggr._key":"latency","aggr._startTime":"2022-12-14T06:39:06.081Z","aggr._sum":1.15}
       ```
 
+### <a name="rate_limiter"></a>
+* `rate_limiter`: Processes the events and controls the number of events aggregated per second. By default, any events exceeding the configured number of events per second are dropped. This behavior can be overwritten with a config option which blocks until the events are allowed.
+    * It supports the following config options
+       * `events_per_second`: Number of events allowed per second
+       * `drop_when_exceeds`: indicates if the events should be dropped when more number of events than the number of events allowed per second are received. Default value is true.
```

Comment:
Please see these comments in your other PR: https://github.com/opensearch-project/data-prepper/pull/2096/files#r1068671493

Reply:
I think with two possible values, true/false is more appropriate than `enum`. I will modify it to default it to false.

Reply:
I still think the enum is better since the user is choosing between two behaviors which are not quite related.

If I read `drop_when_exceeds: false`, I have no clue what actually happens when the messages are exceeded. I have to go to the documentation to find out.

I'm ok to accept with a boolean to make progress though.

Reply:
Regardless, we need to update this documentation to match the changes.